backend/app/checkpoint.py

```diff
@@ -19,8 +25,13 @@ def loads(value: bytes) -> Checkpoint:
 
 
 class PostgresCheckpoint(BaseCheckpointSaver):
-    class Config:
-        arbitrary_types_allowed = True
+    def __init__(
```

Comment:
This change is required since the introduction of the `serde` API in `BaseCheckpointSaver`.